sixtyfps_runtime/corelib/lib.rs

```diff
@@ -21,6 +21,8 @@ only be used with `version = "=x.y.z"` in Cargo.toml.
 */
 #![doc(html_logo_url = "https://sixtyfps.io/resources/logo.drawio.svg")]
 #![deny(unsafe_code)]
+//TODO #![cfg_attr(not(feature = "std"), no_std)]
+extern crate alloc;
```

Comment:
What does the TODO mean?

Reply:
That we need to do this:

```suggestion
#![cfg_attr(not(feature = "std"), no_std)]
extern crate alloc;
```

When the crate can compile without the "std" feature :-)

Reply:
Ah yes, of course - sorry :)